resources/output_strings.js

```diff
@@ -0,0 +1,536 @@
+// TODO: make this a static class with static members
+//       this requires babel-eslint or being patient.
+
+// Throw exceptions on invalid property names to catch easy typos.
+class ThrowOnInvalidProxy {
+  constructor(obj) {
+    this.obj = obj;
+
+    if ('toJSON' in obj)
+      throw new Error('Cannot have toJSON property.');
+
+    return new Proxy(this, {
+      set(target, property, value) {
+        throw new Error('Cannot set readonly object.');
+      },
+
+      get(target, name) {
+        if (name === 'toJSON')
+          return JSON.stringify(obj);
+
+        if (name in target.obj)
+          return target.obj[name];
+
+        throw new Error(`Unknown property ${name}`);
+      },
+    });
+  }
+}
+
+// General guidelines:
+// * property names should closely match English text
+// * use OnPlayer suffix for things with `${player}`
+// * use OnTarget suffix for things with `${name}`
+// * any other parameters (of which there are none, currently) should use consistent suffixes.
+// * the value of each property should be a single object with localized keys
```

Comment:
Feel free to disagree or propose other guidelines.  This is what I thought made sense to me for a set of translations that would get used elsewhere outside of responses.

I do think the indirection causes a little bit of cognitive friction, which is why I proposed the rules of closely match text, and use common suffixes for the two included parameter types.

Reply:
The only one that I feel like bothers me here is `lookAwayFromTarget`, since I feel that one is usually player, but has the corner case of sometimes being look away from boss (which would just be `lookAway`). Is there anything else you have to distinctly look away from?

For context, I'm fine with it staying `lookAwayFromTarget`, but my issue is that it's mostly player and maybe we should just add a line here or something that it OnPlayer is distinctly always player mechanics, or something like that.

I get it largely doesn't matter at the end of the day, but it did make me do a double take in the review if `lookAwayFromTarget` didn't get updated properly.

Reply:
We could arguably have both?

Reply:
Added it.  Do you think `lookAwayFrom` should use both of them? Or fine to just have both in `Outputs`?